lib/core/sessions.js

```diff
@@ -617,6 +617,21 @@ class ServerSessionPool {
   }
 }
 
+// TODO: this should be codified in command construction
+// @see https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#read-concern
+function commandSupportsReadConcern(command) {
+  return (
```

Comment:
how do you feel about this instead:

```js
const READ_CONCERN_COMMANDS = new Set('aggregate', 'count', 'distinct', 'find', 'mapReduce', 'parallelCollectionScan', 'geoNear', 'geoSearch']);
function commandSupportsReadConcern(command) {
  return READ_CONCERN_COMMANDS.has(Object.keys(command)[0]);
}
```

Reply:
I think the way I wrote it is likely much faster, but also this will all go away in about a week. The only reason we need this here is because we're not using operations for these commands. They will be operationalized for retryable reads, so I'm inclined to let this sit for now.